src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/enums/PrisonerSupportedAlertCodeType.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.enums
+
+@Suppress("unused")
+enum class PrisonerSupportedAlertCodeType {
```

Comment:
should the name be something else - like VisitReviewAlertCodes?

Reply:
I think PrisonerSupportedAlertCodeType is okay, as it now will not be used in the DB and is not being stored. Even if we store it, it would be in a table of it's own context... so I think it's okay